src/llvm/tools/clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -42,6 +42,16 @@ def err_drv_cuda_version_unsupported : Error<
   "--no-cuda-version-check.">;
 def err_drv_cuda_host_arch : Error<"unsupported architecture '%0' for host compilation.">;
 def err_drv_mix_cuda_hip : Error<"Mixed Cuda and HIP compilation is not supported.">;
+def warn_drv_msp430_hwmult_unsupported : Warning<"The given MCU doesn't "
+  "support hardware multiplier, but -mhwmult is set to %0.">,
+   InGroup<InvalidCommandLineArgument>;
+def warn_drv_msp430_hwmult_mismatch : Warning<"The given MCU supports %0 "
+  "hardware multiplier, but -mhwmult is set to %1.">,
+   InGroup<InvalidCommandLineArgument>;
+def warn_drv_msp430_hwmult_no_device : Warning<"No MCU device specified, but "
+  "'-mhwmult' is set to 'auto', assuming no hardware multiply. Use -mmcu to "
```

Comment:
multiply => multiplier?

Reply:
Sounds okay to me. I'm even thinking about change to 'multiply' in the other warnings cause 'supports' 'multiplier' looks really weird.